main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/josephroberts/edge-node-manager/application"
+	"github.com/josephroberts/edge-node-manager/config"
+	"github.com/josephroberts/edge-node-manager/database"
+	"github.com/josephroberts/edge-node-manager/device"
+	"github.com/josephroberts/edge-node-manager/device/deviceType"
+	"github.com/josephroberts/edge-node-manager/radio/radioType"
+)
+
+func main() {
+	exit()
+
+	nrf51822 := &device.Type{
+		Device: deviceType.NRF51822,
+		Radio:  radioType.BLUETOOTH,
+	}
+	esp8266 := &device.Type{
+		Device: deviceType.ESP8266,
+		Radio:  radioType.WIFI,
+	}
+
+	applications := make([]*application.Application, 0, 10)
+	applications = append(applications,
+		&application.Application{
+			UUID:       "resin",
+			DeviceType: nrf51822,
+		})
+	applications = append(applications,
+		&application.Application{
+			UUID:       "resin_esp8266",
+			DeviceType: esp8266,
+		})
+
+	if delay, err := config.GetLoopDelay(); err != nil {
```

Comment:
You don't have to nest to get access to `delay`, you can access it in the else branch.

Probably you'd be better off just putting this outside of the `if`.